src/components/camera.js

```diff
@@ -1,6 +1,7 @@
 var registerComponent = require('../core/component').registerComponent;
 var THREE = require('../lib/three');
 var utils = require('../utils/');
+var bind = utils.function.bind;
```

Comment:
If you code trying to anticipate everything you will have super abstract unmaintainable code with optimizations that are there for no reason. Let's just add a bind file and generalize, create a namespace when there's a need.